config/__init__.py

```diff
@@ -265,7 +265,7 @@
 TENTACLE_PACKAGE_DESCRIPTION = "package_description"
 EVALUATOR_CONFIG_FOLDER = "config"
 TENTACLES_TRADING_MODE_PATH = "Mode"
-TENTACLES_DEFAULT_BRANCH = "master"
+TENTACLES_DEFAULT_BRANCH = "dev"
```

Comment:
That was for your tests or we need to keep this before releasing ?

Reply:
We will need to keep this till the next release because DailyTradingMode now uses get_number_of_traded_assets and get_holdings_ratio that exists only in the dev branch of OctoBot => tentacles have to stay in dev otherwise regular users will not be able to use DailyTradingMode

Reply:
Yes ok !